README.md

```diff
@@ -1,70 +1,13 @@
-# article-json-to-contenteditable
+# article-json-to-contenteditable [![Build Status](https://travis-ci.org/micnews/article-json-to-contenteditable.png?branch=master)](https://travis-ci.org/micnews/article-json-to-contenteditable)
+
 
-## Installation
 
-Download node at [nodejs.org](http://nodejs.org) and install it, if you haven't already.
+## Installation
 
 ```sh
 npm install article-json-to-contenteditable --save
 ```
-## Example
```

Comment:
If you move the example/client.js file to the root and name it example.js I think it will be included in the readme automatically. Might be nice, wdyt?